src/plugins/discover/public/application/components/no_results/no_results.tsx

```diff
@@ -229,7 +244,7 @@ export const DiscoverNoResults = ({ queryString, query, savedQuery, timeFieldNam
                   <EuiSpacer size="s" />
                   {sampleQueries
                     .slice(0, 5)
-                    .map((sampleQuery) =>
+                    .map((sampleQuery: any) =>
```

Comment:
why any? can we be more specific?

Reply:
`any` was part of the preexisting logic. It wasn't adhering to any object and was just returning the title and the query. 
Ex: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/76cf823e0a52e101e3e9b5fbe03836ec529229da/src/plugins/data/public/query/query_string/dataset_service/lib/index_type.ts#L94